deep_water_level/src/data.py

```diff
@@ -73,7 +73,7 @@ def parse_line(line):
         data = [parse_line(line) for line in lines]
 
         # Remove any data points with None values
-        data = [d for d in data if all(d[1])]
+        data = [d for d in data if (None not in d[1])]
```

Comment:
is this because 0's won't work with all?

Reply:
Ah yes, exactly. Which i found out because i lazily made an annotations file with `0,0,0,0` for the water line :-)